tests/test_tokenization_common.py

```diff
@@ -4170,32 +4169,52 @@ def test_split_special_tokens(self):
             return
 
```

Comment:
Would probably add a comment here saying why we don't test against explicit values

Reply:
agreed, adding